symbolic-debuginfo/src/macho/mod.rs

```diff
@@ -18,7 +18,7 @@ use crate::private::{MonoArchive, MonoArchiveObjects, Parse};
 
 mod bcsymbolmap;
 
-pub use bcsymbolmap::{BcSymbolMap, BcSymbolMapError};
+pub use bcsymbolmap::{BcSymbolMap, BcSymbolMapError, BitcodeUuidMapping};
```

Comment:
Please make this a star-reexport. It's too easy to miss types here, for example here you forgot the `PListError` which is in the return type of `BitcodeUuidMapping::parse_plist`.

```suggestion
pub use bcsymbolmap::*;
```